### DISCUSSION TITLE
Does SpotifyAPI-NET work on Android?

### DISCUSSION THREAD

**Discussion by dharber** [JohnnyCrazy/SpotifyAPI-NET#789] [Q&A]
Running a MAUI app on an Android device. Nothing generates an error but neither is the code ever receiving an authorization code. It works fine on Windows in a WPF application. I can imagine that the EmbedIOAuthServer might not work properly on Android (although since I assume it's a .NET class, it might at least pretend to work.

Is this expected to work at all on Android or is that just too optimistic?

Thanks!

**Comment by JohnnyCrazy** [comment]
`EmbedIOAuthServer` probably won't work on Android. Didn't test myself yet, but Android is very restricted. On android, it might make more sense to use a custom protocol handler. The UWP example app uses a similar approach.
